src/browser/components/events/ride.jsx

```diff
@@ -61,9 +61,11 @@ Ride = React.createClass({
 
         return (
             <li onClick={this.toggleActive} className='ride-list-item'>
-                <div className='icon icon-avatar ride-icon' />
-                <h2 className='ride-driver'>{ride.driverName}</h2>
-                <h3 className='ride-spots'>{availableSpots} available</h3>
+                <span className='icon icon-avatar ride-icon' />
+                <div className='driver-summary'>
```

Comment:
Pull this out into another div since we want the icon and the div to be side by side. I changed it to a span to match the other icons too.